easysid.cpp

```diff
@@ -38,6 +55,35 @@ EXPORT void end_capture()
     }
 }
 
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        // TODO: Implement errors, or SEH to handle these scenarios.
```

Comment:
This is cross-platform code. "SEH" is a Windows-specific thing. Remove the mention of SEH here and elsewhere.